### ISSUE TITLE
[Battlescape] Incorrect Tile Loop Crash

### ISSUE THREAD

**Issue by FilmBoy84** [OpenApoc/OpenApoc#754]
This crash can sometimes completely CTD the game, often you just have to OK so many dialogues it's often not worth playing any longer

Here's a savegame and the version of the EWM in use at the time
[expanded weapons test 1 mod.zip](https://github.com/OpenApoc/OpenApoc/files/3606611/expanded.weapons.test.1.mod.zip)
[save_Expanded Weapons Test 1.zip](https://github.com/OpenApoc/OpenApoc/files/3606612/save_Expanded.Weapons.Test.1.zip)

If you leave the mission time to go forwards, sooner or later these errors appear

![image](https://user-images.githubusercontent.com/33913088/64803669-259e2a80-d585-11e9-8781-6afb95df99c6.png)
![image](https://user-images.githubusercontent.com/33913088/64803736-53836f00-d585-11e9-82c0-7a81b47983c0.png)

See also;  #587 #581 #718 #393 and #693

**Comment by Paddywhacker** [comment]
BattleUnit pathing in OA just isn't as smooth as in the original, and a lot of it revolves around 'give way'.  Is there a way to fix this, maybe by, on collision, making the moving unit teleport or the stationary unit briefly "no-clip"?

**Comment by emptyVoid** [comment]
The issue should probably be closed, since it's not reproducible after #756.

**Comment by FilmBoy84** [comment]
Closing as requested by emptyVoid
If the problem returns, please create a new issue and reference this as appropriate
